codes/train.py

```diff
@@ -174,16 +174,16 @@ def main():
                     model.test()
 
                     visuals = model.get_current_visuals()
-                    
+
                     sr_img = util.tensor2img(visuals['SR'])  # uint8
                     gt_img = util.tensor2img(visuals['HR'])  # uint8
-                    
+
                     # print("Min. SR value:",sr_img.min()) # Debug
                     # print("Max. SR value:",sr_img.max()) # Debug
-                    
+
```

Comment:
Not sure what happened here.

Reply:
@bscout9956 it might have auto-formatted the file to use different line ending types (CRLF vs LF)

Reply:
I think that would've left a bigger mess. Probably PyCharm stripping tabs, spaces or something.

Reply:
Please remove the whitespace changes

Reply:
So put back the tabs??

Reply:
Changes made!